src/linter/index.tsx

```diff
@@ -0,0 +1,78 @@
+type Message = {
+  column: number | null;
+  description: string | string[];
```

Comment:
It'd be easier to use if we had only `string[]` (the `string |` could be replaced with a single-element array).

Reply:
Ah, yes, this should be easier to work with